be/src/formats/parquet/scalar_column_reader.cpp

```diff
@@ -18,10 +18,28 @@
 #include "gutil/casts.h"
 #include "io/shared_buffered_input_stream.h"
 #include "simd/simd.h"
+#include "statistics_helper.h"
 #include "utils.h"
 
 namespace starrocks::parquet {
 
+StatusOr<bool> FixedValueColumnReader::row_group_zone_map_filter(const std::vector<const ColumnPredicate*>& predicates,
+                                                                 CompoundNodeType pred_relation,
+                                                                 const uint64_t rg_first_row,
+                                                                 const uint64_t rg_num_rows) const {
+    ZoneMapDetail zone_map{_fixed_value, _fixed_value, _fixed_value.is_null()};
+    auto is_satisfy = [&](const ZoneMapDetail& detail) {
+        if (pred_relation == CompoundNodeType::AND) {
```

Comment:
I have a sense that probably we should not write in this way?

Because `pred_relation` should be handled in zone_map_filetr_evaluator, but not here.
- zonemap_filter_evaluator can pass each predicate to column reader
- and zonemap_filter_evaluator can do AND/OR stuff and short circuit stuff.